testbom/pom.xml

```diff
@@ -40,6 +40,8 @@
     <name>WildFly: Core BOM of Test Dependencies</name>
 
     <properties>
+        <version.com.google.guava>30.1.1-jre</version.com.google.guava>
```

Comment:
This is expected to be 31.1-jre, introduced by https://github.com/wildfly/wildfly-core/pull/5237

In addition, you have to remove from the main pom.xml the properties of the artifacts you are moving to test bom:
```
<version.com.google.guava>31.1-jre</version.com.google.guava>
<version.com.google.guava.failureaccess>1.0.1</version.com.google.guava.failureaccess>
```

Reply:
Made the changes. Thank you. Please review.